### DISCUSSION TITLE
希望能出个支持预览OFD或ODF文件的内置控件

### DISCUSSION THREAD

**Discussion by joyoes** [AvaloniaUI/Avalonia#15590] [Q&A]
### Is your feature request related to a problem? Please describe.

像一些OFD或PDF文件比较大，如果转换成图片再显示出来，性能实在太差了！希望能出个直接预览PDF和ODF文件的内置控件

### Describe the solution you'd like

像一些OFD或PDF文件比较大，如果转换成图片再显示出来，性能实在太差了！希望能出个直接预览PDF和ODF文件的内置控件

**Comment by timunie** [comment]
only english issues can be accepted. Sorry. I'll convert this into a Q&A

**Comment by thevortexcloud** [comment]
How are you converting the PDFs to images? You likely don't need to make them very high resolution anyway.
